README.md

```diff
@@ -157,7 +157,7 @@ Encountering an issue? Before reaching out to support, we recommend heading to o
 
 We welcome all contributors, from casual to regular 💙
 
-- **Bug report**. Is something not working as expected? [Send a bug report](https://github.com/algolia/instantsearch.js/issues/new?template=Bug_report.md).
+- **Bug report**. Is something not working as expected? [Send a bug report](https://github.com/algolia/instantsearch.js/issues/new?template=BUG_REPORT.yml&labels=triage,Package%3A%20instantsearch.js).
```

Comment:
Do we need to specify "triage" if it's already in the template? Does the query parameter override it entirely instead of merging?

Reply:
Indeed, labels specified in URL search parameters will override the ones in the issue form config file.

Reply:
I retested it and now it seems the triage label is applied just after page load. I'll remove it from the search parameters 😅 .